eemployee_storage/employee-storage.sh

```
@@ -0,0 +1,43 @@
+
+echo "Region: $REGION"
+echo "VPC Name: $VPC_NAME"
+echo "Bucket Name: $BUCKET_NAME"
+echo "Creating s3 buckets..."
+
+
+# Creating VPC - Got the VPC ID from localstack chatbot
+
+
+echo "Creating VPV..."
+VPC_ID=$(aws ec2 create-vpc \
```

Shellcheck (0.10.0)

[warning] VPC_ID appears unused. Verify use (or export if used externally).

(SC2034)